File: restaurant-frontend/src/Component/Services/CommentService.js
import axios from "axios";

const COMMENT_BASE_REST_API_URL= 'http://localhost:8080/api/comment';

class CommentService{

    getAllComments(){
        return axios.get(COMMENT_BASE_REST_API_URL);
    }
    createComment(comment){
        return axios.post(COMMENT_BASE_REST_API_URL,comment);
    }
   
}

export default new CommentService();

File: restaurant-backend/src/main/java/com/app/restaurant/service/CommentService.java
package com.app.restaurant.service;

import com.app.restaurant.model.Comments;
import com.app.restaurant.repository.CommentRepository;
import org.hibernate.mapping.Column;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
public class CommentService {
    private CommentRepository commentRepo;

    public Comments createComment(String comment){
        Comments newComment = new Comments();
        newComment.setComment(comment);
        commentRepo.save(newComment);

        return newComment;
    }

    public List<Comments> getAllComments(){
        return commentRepo.findAll();
    }
}
